utils/api.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+export default axios.create({
+    baseURL:"https://elite-bn-be.herokuapp.com/",
```

Comment:
this url should be from an environment variable.
also remove the `/` it will be added by the person writting an api call

Reply:
done , i added `/` to test if it is working